longitudinal/app.R

```diff
@@ -154,6 +154,9 @@ var_labels(ADSL) <- c(adsl_labels)
 # other values can be adjusted for as well of course but this is study specific pre-processing that needs to be done
 
 # adjust select based on your study specifics
+
+set.seed(1, kind = "Mersenne-Twister") # Reproducible code due to `sample` calls
```

Comment:
Why this? Is there any randomness?

Reply:
Yeah, we take `sample()` in two places within the `ADLB_SUBSET <- ADLB`.
And, I confirm that this fix works.

Reply:
good catch 👍